spec/forms/search_developer_form_spec.rb

```diff
@@ -0,0 +1,84 @@
+
+require 'rails_helper'
+
+RSpec.describe SearchDeveloperForm do
+  describe '#search' do
+    before do
+      @language1 = create(:language, code: 'vn')
```

Comment:
@thupv 
<img width="699" alt="screen shot 2018-04-04 at 14 11 55" src="https://user-images.githubusercontent.com/4110469/38295052-fd432a16-3816-11e8-9a0d-3f677b7beac5.png">

Reply:
I fixed that problem but github don't recognise, could you check here: https://github.com/thupv/FindDeveloper/pull/1/files